pyEpiabm/pyEpiabm/tests/test_unit/test_sweep/test_travel_sweep.py

```diff
@@ -0,0 +1,138 @@
+import unittest
+import numpy as np
+
+import pyEpiabm as pe
+from pyEpiabm.core import Parameters
+from pyEpiabm.sweep import TravelSweep
+from pyEpiabm.property import InfectionStatus
+from pyEpiabm.tests.test_unit.parameter_config_tests import TestPyEpiabm
+
+
+class TestTravelSweep(TestPyEpiabm):
+    """Test the 'TravelSweep' class.
+    """
+
+    def setUp(self) -> None:
+        super(TestTravelSweep, self).setUp()
+        self.travelsweep = TravelSweep()
+
+        # Construct a population with 20 individuals in 1 cells,
+        # 2 microcells and 4 households with 2 infectors.
+        self._population = pe.Population()
+        self._population.add_cells(1)
+        self.cell = self._population.cells[0]
+        self._population.cells[0].add_microcells(2)
+        self.microcell1 = self.cell.microcells[0]
+        self.microcell2 = self.cell.microcells[1]
+        self.microcell1.add_people(15)
+        self.microcell2.add_people(5)
+        self.microcell1.add_household(self.microcell1.persons.copy()[:10])
+        self.microcell1.add_household(self.microcell1.persons.copy()[10:])
+        self.microcell2.add_household(self.microcell1.persons.copy()[:3])
+        self.microcell2.add_household(self.microcell1.persons.copy()[3:])
+        # By default all susceptible, make 2 infectious
+        self.initial_infected_person1 = self.microcell1.persons[0]
+        self.initial_infected_person1.update_status(InfectionStatus(4))
+        self.initial_infected_person2 = self.microcell2.persons[0]
+        self.initial_infected_person2.update_status(InfectionStatus(4))
+
+        self.travelsweep.bind_population(self._population)
+
+    def test_bind_population(self):
+        self.assertEqual(len(self.travelsweep.
+                             travel_params), 3)
+        self.test_sweep = pe.sweep.TravelSweep()
+        self.test_sweep.bind_population(self._population)
+        self.assertEqual(self.test_sweep._population.cells[0]
+                         .persons[0].infection_status,
+                         pe.property.InfectionStatus.InfectMild)
+
+    def test__call__(self):
```

Comment:
Would be great to have a short comment/docstring (doesn't need to be formal) just to make it a bit clearer what is being tested here - in general all the hardcoded values make it harder to read

Reply:
I wrote docstrings for all the tests. Is that helpful?

Reply:
That's great! And we check that the number of people in the population at each point to check it is as expected? Might be good to mention that